defs.bzl

```diff
@@ -242,10 +250,14 @@ _sonarqube = rule(
 def sonarqube(
         name,
         project_key,
-        scm_info,
+        organization_key = None,
+        project_description = None,
+        scm_info = None,
```

Comment:
Making scm_info optional is probably not intended.  Just move it before the arguments with defaults.